### DISCUSSION TITLE
Public gpg key BBB723FB7D4BDD8947EBD95B81C63AE1C224182B

### DISCUSSION THREAD

**Discussion by eikemeier** [renovatebot/renovate#12950] [Request Help]
Renovate Bot on GitHub signes its commits with RSA key BBB723FB7D4BDD8947EBD95B81C63AE1C224182B.

How do I get the public key, so I can verify commit messages?

https://pgp.mit.edu/pks/lookup?search=BBB723FB7D4BDD8947EBD95B81C63AE1C224182B

**Comment by rarkins** [comment]
https://github.com/renovate-bot.gpg

**Reply by eikemeier** [reply]
Thanks.
